cloudformation/stackcopy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash 
+echo Enter Date in from yyyy-dd-mm
```

Comment:
I'm slightly worried about this manual step, since this date might overwrite the templates in the already existing folder.

Can you get the date from the computer then put it into that format (yyyy-dd-mm) + a timestamp to avoid folder collisions? so it would be something like yyyy-dd-mm-ms (ms for milliseconds -- if "ms" is not terribly long)